lib/apartment/active_record/postgres/schema_dumper.rb

```diff
@@ -0,0 +1,9 @@
+# This patch prevents `create_schema` from being added to db/schema.rb as schemas are managed by Apartment
+# not ActiveRecord like they would be in a vanilla Rails setup.
+
+require "active_record/connection_adapters/postgresql/schema_dumper"
```

Comment:
Should this file be included in this PR?

Reply:
Yeah, it's to make sure that the class is loaded before we monkey-patch it

Reply:
Right, I'm not seeing the file related to `active_record/connection_adapters/postgresql/schema_dumper`.

Reply:
It's in rails source